libmscore/fret.cpp

```diff
@@ -305,7 +316,7 @@ void FretDiagram::draw(QPainter* painter) const
       painter->setFont(scaledFont);
 
       // dotd is the diameter of a dot
-      qreal dotd = stringDist * .7;
+      qreal dotd = _spatium * .49 * score()->styleD(Sid::fretDotSize);
```

Comment:
.49?

Reply:
Also, unless I'm missing something, there is no way to set freDotSize, or the other two new style settings?

Reply:
_stringDist used to be _spatium * .7